test/reducers/language_spec.js

```diff
@@ -0,0 +1,36 @@
+import {expect} from 'chai';
+import deepFreeze from 'deep-freeze';
+
+import reducer from '../../src/reducers/language';
+
+describe('language reducer', () => {
+  describe('SET_LANGUAGE', () => {
+    it('sets current language to argument', () => {
+      const initialState = 'sweden';
+      const action = {
+        type: 'SET_LANGUAGE',
+        payload: 'norway'
+      };
+
+      deepFreeze(initialState);
+      deepFreeze(action);
+      const nextState = reducer(initialState, action);
+
+      expect(nextState).to.equal('norway');
+    });
+
+    it('sets current language to null if argument is not spesified or not valid', () => {
```

Comment:
Tenker du på at "ikke spesifisert" betyr en tom string, eller at den ikke er definert?

I denne testen er payload definert (defined) som en tom string. For å teste når språk ikke er definert (undefined), så kan du ha `payload: undefined`, eller bare fjerne payload fra action.

Hvis det er for å teste en tom string, så burde det kanskje stå inne i `it`: `sets current language to null if argument is an empty string`.
Og så lage en egen test for språk som ikke er gyldig/godkjent, feks:
`it('sets current language to null if argument is not in list of valid languages')`.
Poenget er at hver test skal teste én ting, og at det som står inne i `it` skal beskrive hva funksjonen gjør (forventet output ved en gitt input).